src/providers/data_provider_be.c

```diff
@@ -110,9 +110,28 @@ static int get_offline_timeout(struct be_ctx *ctx)
     return offline_timeout;
 }
 
+static int get_offline_timeout_max(struct be_ctx *ctx)
+{
+    int offline_timeout_max;
+    errno_t ret;
+
+    ret = confdb_get_int(ctx->cdb, ctx->conf_path,
+                         CONFDB_DOMAIN_OFFLINE_TIMEOUT_MAX, 3600,
+                         &offline_timeout_max);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_CONF_SETTINGS,
+              "Failed to get offline_timeout_max from confdb. "
+              "Will use 3600 seconds.\n");
+        offline_timeout_max = 3600;
```

Comment:
I think that changing those hardcoded "60" and "3600" everywhere may be a good candidate for next contribution.
They may look better as a some function local const or global define if it is worht it. Right now in future when someone change 60s to lets say 90s will have to go around multiple lines to fix it.

Reply:
Sounds good, I've added some defines at the top of the file. Apologies for the delay getting back to this PR.